_15thnight/api/alert.py

```diff
@@ -97,6 +105,7 @@ def delete_alert(id):
     """
     Delete an alert.
     """
+    return 'Not Implemented', 501 # We do not support a UI for this
     if current_user.role == 'advocate':
```

Comment:
This was added for security concerns. The code is left because we may want to have the ability to delete alerts in the future, but we don't currently support it. Until we finalize the spec, it is not implemented.